soaks/soak.sh

```diff
@@ -99,25 +99,64 @@ for SOAK_NAME in ${SOAKS}; do
     echo "########    ${SOAK_NAME}"
     echo "########"
     echo "########"
+    capture_dir_replica_one="${capture_dir}/1"
```

Comment:
👍 to running multiple runs locally too.

Nit: is there a reason we couldn't have used a loop?

Reply:
Above 3 is when I tend to use a loop in bash, considering how weird they are to get right. But, no, a loop would work here too.

Reply:
Love that reasoning. Bash language designers: take note 😆